tests/plugins/print/manual/print.md

```diff
@@ -0,0 +1,11 @@
+@bender-ui: collapsed
+@bender-tags: 3661, 4.14.0, feature
+@bender-ckeditor-plugins: wysiwygarea, print, font, colorbutton, format, clipboard, pagebreak, toolbar, floatingspace, link, image2
+
+Things to check:
+
+* There are both "Preview" and "Print" buttons in the toolbar.
+* Both "Preview" and "Print" commands are disabled in source mode.
+* "Print" button prints the same document as in preview. **Note**: preview shows pagebreak element, in all browsers except IE8, as it is not possible to render new page in HTML. However this element should not be visible on printed document.
+* Preview is closed after confirming/dismissing native print dialog.
+* Check if page is rendered correctly inside native print dialog's preview and in printed content.
```

Comment:
It's a bit hard to follow. Maybe change it into proper test steps, so they are simpler to reproduce? Currently seems more like a generic test.